### ISSUE TITLE
gather

### ISSUE THREAD

**Issue by ZTurboX** [PaddlePaddle/Paddle#39637]
paddle中的gather怎么使用？总是报错。比如在pytorch中是这样写的`a = torch.Tensor([[1,2,3,4,5,6], [0,1,2,3,4,5]])
b = a.gather(1, torch.tensor([[5,4,3,2,1,0],[0,1,2,3,4,5]]))`，怎么转换成paddle的写法

**Comment by pangyoki** [comment]
你好，可以用`take_along_axis`这个API：https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/take_along_axis_cn.html#take-along-axis

<img width="322" alt="图片" src="https://user-images.githubusercontent.com/26408901/154284305-97a15759-7de1-4c5f-af6f-9851ab8e93e2.png">

但是这个API在paddle2.2正式版还没有发布，你可以在官网下载develop的包来使用这个API：https://www.paddlepaddle.org.cn/install/quick?docurl=/documentation/docs/zh/develop/install/pip/linux-pip.html

**Comment by m828** [comment]
> 你好，可以用`take_along_axis`这个API：https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/take_along_axis_cn.html#take-along-axis
> 
> <img alt="图片" width="322" src="https://user-images.githubusercontent.com/26408901/154284305-97a15759-7de1-4c5f-af6f-9851ab8e93e2.png">
> 
> 但是这个API在paddle2.2正式版还没有发布，你可以在官网下载develop的包来使用这个API：https://www.paddlepaddle.org.cn/install/quick?docurl=/documentation/docs/zh/develop/install/pip/linux-pip.html

很奇怪，我尝试换成这个，能够部署在Windows下，在Linux下跑的模型不能够用tensorrt加速推理，将其权重搬到Windows下，可以推理成功，但是Windows不支持benchmark，不能够计算fps

**Comment by m828** [comment]
两边都是2.3.2.post112，tensorrt为8.2.5
